.github/workflows/on_release.yml

```diff
@@ -163,3 +169,110 @@ jobs:
           tags: |
             permitio/opal-client-cedar:latest
             permitio/opal-client-cedar:${{ env.opal_version_tag }}
+
+      - name: Python setup
+        if: github.event_name == 'release' && github.event.action == 'created'
+        uses: actions/setup-python@v5
+        with:
+          python-version: '3.11.8'
+
+      # This is the root file representing the package for all the sub-packages.
+      - name: Bump version - packaging__.py
+        if: github.event_name == 'release' && github.event.action == 'created'
+        run: |
+          version_tag=${{ github.event.release.tag_name }}
+          version_tuple=$(echo $version_tag | sed 's/\./, /g')
+          sed -i "s/VERSION = (.*/VERSION = (${version_tuple})/" packages/__packaging__.py
+          cat packages/__packaging__.py
+          git config --local user.email "eli@permit.io"
+          git config --local user.name "elimoshkovich"
+          git add packages/__packaging__.py
+          git commit -m "Bump version to ${version_tag}"
+
+      - name: Cleanup setup.py and Build every sub-packages
+        if: github.event_name == 'release' && github.event.action == 'created'
+        run: |
+          pip install wheel
+          cd packages/opal-common/ ; rm -rf *.egg-info build/ dist/
+          python setup.py sdist bdist_wheel
+          cd ../..
+          cd packages/opal-client/ ; rm -rf *.egg-info build/ dist/
+          python setup.py sdist bdist_wheel
+          cd ../..
+          cd packages/opal-server/ ; rm -rf *.egg-info build/ dist/
+          python setup.py sdist bdist_wheel
+          cd ../..
+
+      # Upload package distributions to the release - Assets
+      - name: Upload assets to release - Opal-Common
```

Comment:
That's super cool!
But I suspect that wouldn't work.
1. You use a single `asset_name` but the source path (`asset_path`) is a wildcard (that would expand to 2 files).
2. Even if It would work - those 2 files (`.whl` & `.tar.gz`) should be uploaded as assets separately.
3. It seems like that action is not maintained anymore.
4. Nowhere here have you mentioned the name of the release for which the assets should be uploaded. In their documentation they mention `This Workflow example assumes you have the @actions/create-release Action in a previous step`